test/test_coverage.sh

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+# Create a link for the code to current dir
+ln -s src/main/scala/org ./
```

Comment:
I meant you don't have to explain each line in English.  It's much more useful to write the non-obvious: why you do a ln and rm around sbt coveralls and what other dependencies are there.

Btw should the `sbt coverageAggregate` be moved here?  Right before it's submitted?  And the file name is a bit misleading because it's not collecting coverage but only submitting, right?